### DISCUSSION TITLE
can i use query parameter be directly concatenated onto the path when using the router. push() method?

### DISCUSSION THREAD

**Discussion by 18257552642** [vuejs/router#1808] [Help and Questions] [answered]
For example :   router. push ({path: "/abc", query: {flag: 1}) 
be replaced  :   router. push ({path: "/abc? flag=1")

Route successfully redirected, but no "? flag=1" after "/abc", just "/abc" in the browser address bar

**Comment by posva** [comment, marked as answer]
Pass a string as the argument: `router.push('/abc?flag=1')`
